modules/apps/batch-planner/batch-planner-rest-test/src/testIntegration/java/com/liferay/batch/planner/rest/resource/v1_0/test/SiteScopeResourceTest.java

```diff
@@ -170,8 +179,18 @@ private void _testGetPlanInternalClassNameKeySiteScopesPage(
 		}
 
 		for (Group group : expectedGroups) {
+			String groupLabel;
+
+			if (Objects.equals(group.getDescriptiveName(), "Global")) {
```

Comment:
Which are the values returned for `getDescriptiveName` and `getGroupKey` in both cases? I ask because I don't know and this if confuses me a bit

Reply:
for Global object:
- getGroupKey is `"20121"`
- getDescriptionName is `"Global"`

Reply:
And for the general case?

Reply:
for General case:
- getGroupKey is the id of the group.
- getDescriptionName is the name of the group.